ironic_tempest_plugin/tests/scenario/ironic_standalone/test_basic_ops.py

```diff
@@ -443,104 +443,170 @@ def test_ip_access_to_server(self):
+class BaremetalIdracWSManDeployNodepxe(bsm.BaremetalStandaloneScenarioTest):
+
+    mandatory_attr = ['driver']
+    api_microversion = '1.31'  # to set the deploy_interface
+    driver = 'idrac'
+    image_ref = CONF.baremetal.whole_disk_image_ref
+    boot_interface = 'pxe'
 
     @classmethod
     def skip_checks(cls):
-        super(BaremetalIdracWSManDeployNodeipxe, cls).skip_checks()
+        super(BaremetalIdracWSManDeployNodepxe, cls).skip_checks()
         if CONF.baremetal_feature_enabled.ipxe_enabled:
-            skip_msg = ("Test covered with boot interface as ipxe")
+            skip_msg = ("Skipping the Test case since ipxe is enabled")
             raise cls.skipException(skip_msg)
-    boot_interface = 'pxe'
+
+    @utils.services('image', 'network')
+    @decorators.idempotent_id('20c0f2f4-723b-4d1e-a4f2-bea11ffb1513')
+    def test_deploy_node(self):
+        self.boot_and_verify_node()
 
 
 class BaremetalIdracRedfishDeployNode(bsm.BaremetalStandaloneScenarioTest):
 
     mandatory_attr = ['driver']
-
     api_microversion = '1.31'  # to set the deploy_interface
     driver = 'idrac'
     image_ref = CONF.baremetal.whole_disk_image_ref
     boot_interface = 'ipxe'
     power_interface = 'idrac-redfish'
     management_interface = 'idrac-redfish'
 
-    @decorators.idempotent_id('d2bebda2-ad27-4b10-9015-f7d877f0eb87')
+    @classmethod
+    def skip_checks(cls):
+        super(BaremetalIdracRedfishDeployNode, cls).skip_checks()
+        if not CONF.baremetal_feature_enabled.ipxe_enabled:
+            skip_msg = ("Skipping this TC since ipxe is disabled")
+            raise cls.skipException(skip_msg)
+
     @utils.services('image', 'network')
-    def test_ip_access_to_server(self):
+    @decorators.idempotent_id('cd2db251-8a98-4fe7-88ae-a7c0fc8bb786')
+    def test_deploy_node(self):
         self.boot_and_verify_node()
 
 
-class BaremetalIdracRedfishDeployNodepxe(BaremetalIdracRedfishDeployNode):
+class BaremetalIdracRedfishDeployNodepxe(bsm.BaremetalStandaloneScenarioTest):
+
+    mandatory_attr = ['driver']
+    api_microversion = '1.31'  # to set the deploy_interface
+    driver = 'idrac'
+    image_ref = CONF.baremetal.whole_disk_image_ref
+    boot_interface = 'pxe'
+    power_interface = 'idrac-redfish'
+    management_interface = 'idrac-redfish'
 
     @classmethod
     def skip_checks(cls):
         super(BaremetalIdracRedfishDeployNodepxe, cls).skip_checks()
         if CONF.baremetal_feature_enabled.ipxe_enabled:
-            skip_msg = ("Test covered with boot interface as ipxe")
+            skip_msg = ("Skipping the Test case since ipxe is enabled")
             raise cls.skipException(skip_msg)
-    boot_interfaces = 'pxe'
+
+    @utils.services('image', 'network')
+    @decorators.idempotent_id('c992a601-0eec-44d8-9ddf-ffc2f3c4c8f7')
+    def test_deploy_node(self):
+        self.boot_and_verify_node()
 
 
 class BaremetalRedfishDeployNode(bsm.BaremetalStandaloneScenarioTest):
```

Comment:
can be removed as its duplicating existing test case : https://github.com/openstack/ironic-tempest-plugin/blob/f904d287a83f1f77a9be400f51f98bc7eb3b1d7c/ironic_tempest_plugin/tests/scenario/ironic_standalone/test_basic_ops.py#L424

Reply:
Done